isis/tests/FunctionalTestsAutomos.cpp

```diff
@@ -0,0 +1,187 @@
+#include "automos.h"
+
+#include "Fixtures.h"
+#include "TestUtilities.h"
+#include "UserInterface.h"
+#include "Histogram.h"
+
+#include "gmock/gmock.h"
+
+using namespace Isis;
+
+static QString APP_XML = FileName("$ISISROOT/bin/xml/automos.xml").expanded();
+
+TEST_F(DefaultCube, FunctionalTestAutomosDefault) {
+  QString singleCubeListPath = tempDir.path() + "/newCubeList.lis";
+  FileList singleCubeList;
+  singleCubeList.append(projTestCube->fileName());
+  singleCubeList.write(singleCubeListPath);
+  QString outPath = tempDir.path() + "/mosaic.cub";
+
+  QVector<QString> args = {"fromlist=" + singleCubeListPath, "mosaic=" + outPath};
+  UserInterface options(APP_XML, args);
+  Pvl appLog;
```

Comment:
Since we don't use the log in the tests and it can be null in the automos call, we can we remove the log variables?